src/Model/BSONDocument.php

```diff
@@ -50,11 +50,9 @@ public function __clone()
      * by default.
      *
      * @see https://php.net/arrayobject.construct
-     * @param array   $input
-     * @param integer $flags
-     * @param string  $iteratorClass
+     * @param array $input
```

Comment:
This looks like it can be removed.

Reply:
Removed this and a number of other superfluous type annotations.